esp/src/eclwatch/PackageMapPartsWidget.js

```diff
@@ -91,11 +91,15 @@ define([
              this.addPartsDropDown.on("apply", function (evt) {
                 if (context.addPartsDropDown.filterForm.validate()){
                     var addPartInput = context.getFilter();
+                    var packageMapSearch = context.params.packageMap.search("::");
+                    var packageMapClean;
+
+                    packageMapSearch > -1 ? packageMapClean = context.params.packageMap.split('::')[1] : packageMapClean = context.params.packageMap;
+
```

Comment:
@GordonSmith I only add one, and it's always in the form of "<target>::".  People usually use the file name for the rest of the ID.  I probably SHOULD have disallowed "::" in the ID itself, but as it is I suppose someone could override the name to use :: as part of the Id. 

You know the target, so the solution that is probably most correct is to look for a leading "<target>::" and remove it if its there.  Probably do case insensitive compare just to be safe.